lib/matplotlib/tests/test_backend_bases.py

```diff
@@ -3,7 +3,7 @@
 from matplotlib import path, transforms
 from matplotlib.backend_bases import (
     FigureCanvasBase, KeyEvent, LocationEvent, MouseButton, MouseEvent,
-    NavigationToolbar2, RendererBase)
+    NavigationToolbar2, RendererBase, VectorizedGraphicsContextBase)
```

Comment:
Don't need this anymore.
```suggestion
    NavigationToolbar2, RendererBase)
```